flashbotsextra/database.go

```diff
@@ -32,12 +34,17 @@ func (NilDbService) GetPriorityBundles(ctx context.Context, blockNum int64, isHi
 	return []DbBundle{}, nil
 }
 
+func (NilDbService) GetLatestUuidBundles(ctx context.Context, blockNum int64) ([]types.LatestUuidBundle, error) {
+	return []types.LatestUuidBundle{}, nil
+}
+
 type DatabaseService struct {
 	db *sqlx.DB
 
-	insertBuiltBlockStmt    *sqlx.NamedStmt
-	insertMissingBundleStmt *sqlx.NamedStmt
-	fetchPrioBundlesStmt    *sqlx.NamedStmt
+	insertBuiltBlockStmt      *sqlx.NamedStmt
+	insertMissingBundleStmt   *sqlx.NamedStmt
+	fetchPrioBundlesStmt      *sqlx.NamedStmt
+	fetchGetLatestUuidBundles *sqlx.NamedStmt
```

Comment:
Nitpicking :  to make it consistent 
```suggestion
	fetchGetLatestUuidBundlesStmt *sqlx.NamedStmt
```